users/models.py

```diff
@@ -145,3 +145,16 @@ class SlackToken(models.Model):
 
     def __str__(self):
         return f"({self.user}) {self.token}"
+
+class TimeLog(models.Model):
+    """ records time in and time out of user
+    """
+
+    user = models.ForeignKey(User, null=True, on_delete=models.SET_NULL)
+    time_in = models.DateTimeField(null=True, blank=True)
```

Comment:
set this to `auto_now_add=True`. it will save you a few lines of codes.

by the way, this is the perfect situation to use the auto_now_add attribute, django made it to be use in this kind of situation (adding timestamp)